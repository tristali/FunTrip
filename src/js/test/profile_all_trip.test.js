import React from "react";
import ProfileAllTrip from "../components/profile/profile_all_trip";
import renderer from "react-test-renderer";

test("AllTrip changes the class when onClick", () => {
    const component = renderer.create(
        <ProfileAllTrip
            name="aaa"
            start="2018/10/31"
            end="2018/11/20"
            id="-LPVHoOOx1ML8iJzLzki"
            key="all_trip_1"
            // state={this.props.state}
            handleAppStateChange={() => {}}
            thisTripState="triping"
        />
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();

    // tree.props.onClick();
    // tree = component.toJSON();
    // expect(tree).toMatchSnapshot();
});
