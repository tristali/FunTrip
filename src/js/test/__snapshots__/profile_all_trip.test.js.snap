// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AllTrip changes the class when onClick 1`] = `
<li
  className="triping"
>
  <div
    onClick={[Function]}
  >
    <ul>
      <li>
        aaa
      </li>
      <li>
        2018/10/31 - 2018/11/20
      </li>
      <li>
        <div />
      </li>
    </ul>
  </div>
</li>
`;
